lib/verdict/experiment.rb

```diff
@@ -276,4 +285,16 @@ def is_scheduled?
     end
     return true
   end
+
+  def is_stop_new_assignments?
+    if @schedule_stop_new_assignment_timestamp and
+        @schedule_start_timestamp and
+        @schedule_end_timestamp and
```

Comment:
Any reason why the `schedule_stop_new_assignment_timestamp` would need the start date and end date to be present?

Reply:
🤔 Just want to be extra safe here. If the user didn't input the start or end timestamp here, we wouldn't want the assignments to be stopped right?

Reply:
Our contract is just 
> If you have provided a "stop_new_assignments_date", then any date beyond that point we won't make new assignments.

Likewise for start and end date. So I think all 3 of these options can function independently. What do you think?

Reply:
That seems reasonable. With the documentation (comment in the code), I think we are good. Will make the change then 👍